scripts/update_common_constants.rb

```diff
@@ -15,7 +15,7 @@
 GIT_DESCRIBE_VERSION = CUSTOM_VERSION || `git describe --tags | sed s%^dev/%% | sed s/-g.*$// | sed -E "s/-([0-9]*)$/-dev.\\1/"`.rstrip
 GIT_DESCRIBE_VERSION.sub!(%r{^v}, '')
 
-GIT_COMMIT = `git describe --always`.rstrip
+GIT_COMMIT = `git rev-parse --short HEAD`.rstrip
```

Comment:
The previous approach doesn't work correctly on tagged commits. The new approach was suggested here on SO: https://stackoverflow.com/questions/949314/how-to-retrieve-the-hash-for-the-current-commit-in-git#comment17766441_949391